vsts.yml

```diff
@@ -1,16 +1,11 @@
-resources:
-- repo: self
-phases:
-- phase: Build_Electron_via_GN
-  queue:
-    timeoutInMinutes: 180
+jobs:
+- job: Build_Electron_via_GN
+  timeoutInMinutes: 180
   steps:
   - bash: |
-      git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git "${AGENT_BUILDDIRECTORY}/depot_tools"
-      echo "##vso[task.setvariable variable=PATH]$PATH:${AGENT_BUILDDIRECTORY}/depot_tools"
-    name: Setup_depot_tools
-
-  - bash: |
+      export PATH="$PATH:/Users/electron/depot_tools"
```

Comment:
If `depot_tools` is installed in the base image, can we also set up the `PATH` environment variable in the base image?

Reply:
Apparently VSTS does something to the path, because I did set it up in the base image, but it looks like VSTS gets a different path.